### ISSUE TITLE
Font size adjustment

### ISSUE THREAD

**Issue by Seirdy** [enafore/enafore#17]
Enafore uses a variety of font sizes, which causes problems for some users.

Users with display scaling, increased default font sizes, or mobile display zooming see larger text, as intended. Selected posts have even larger text, which means that a single line can fit very few words. Zooming out (something hard to do on full-screen PWAs) fixes the problem for the active post, but makes other text (timestamps, the text in settings screens, etc) excessively small.

A setting to normalize font sizes across the app would make it easier to use large text across the app without seeing gigantic post text.

I imagine that some of Enafore's charm comes from magnified post and active-post text so I don't think the *default* needs to change.

**Comment by easrng** [comment]
Yeah I should make the font sizes and padding more consistent... And I agree that the larger active post text can be an issue sometimes.
